understanding/21/reflow.html

```diff
@@ -16,66 +16,308 @@ <h2>In brief</h2>
     </section>
 
-    <section id="benefits">
+    <section>
+      <h2>Overlap with other success criteria</h3>
+      <p>When reviewing a web page for Reflow, issues related to other WCAG success criteria might be discovered. The following are examples of potential overlaps with other success criteria, though not all overlapping success criteria are mentioned.</p>
+
+      <h3>Resize Text</h3>
+      <p>The focus of the Reflow success criterion is to ensure that users are not prevented from zooming into web pages, and when doing so text within sections of content will wrap so that the text can be read without having to scroll in two directions. <a href="resize-text">Success criterion 1.4.4 Resize Text</a> overlaps with this goal, as it should be possible to increase the size of all text to at least 200% while simultaneously meeting the reflow requirement. For many web pages, the text of the page is expected to continue to enlarge as the browser magnification increases, so that users can magnify text up to (and beyond) 400%. In an implementation where text does not consistently increase its size as people zoom in (such as when it is transformed based on a media query to adapt to small-screen usage), it must still be possible to get to 200% enlargement to satisfy the Resize Text criterion. It is important to note that while the intent of Reflow will commonly result in the enlargement of text, Reflow does not specifically require any text size enlargement be reached. There may even be instances where one needs to prevent some text from becoming too large, and thus requiring unwanted two-dimensional scrolling.</p>
+      
+      <p>For example, if at the default browser setting of 100% zoom, text is set at 20px when the window is 1280 CSS pixels wide, at 200% zoom it will visually appear at twice the size. After zooming by 400% the page reflows to fit within the 320 CSS pixel viewport, the author may decide to set the page's text size to 10px. The text is now half the original size in CSS pixels, but as it has been enlarged to 400%, so it is still displayed at twice the size compared to the default browser setting of 100% zoom. It is not required to achieve 200% text enlargement while remaining inside a specific breakpoint (as zooming may result in the variation for a new breakpoint becoming active), but it should still be possible to get 200% text enlargement in some way compared to the default 100% zoom.</p>
+
```

Comment:
```suggestion
      <p>On small-screened devices (e.g., a device with a 320 CSS pixel display), it is acceptable for a Reflow-conformant page to introduce two-dimensional scrolling when zoomed to 200%, in support of Resize Text.</p>
```

The “[Overlap with other success criteria](https://deploy-preview-4055--wcag2.netlify.app/understanding/reflow#overlap-with-other-success-criteria)” section says “_it should be possible to increase the size of all text to at least 200% while simultaneously meeting the reflow requirement._”

Although this is certainly true for tablet/desktop-ish displays, 2-dimensional scrolling may still necessarily occur at 200% zoom on narrower displays. And this is fine, right?

For example, if I view the [G302 card carousel demo](https://deploy-preview-4055--wcag2.netlify.app/working-examples/reflow-carousel-panel-horizontal-scroll/) at 320px and zoom to 200%, I _**do**_ need to scroll in 2 dimensions to understand a single card in its entirety. And yet Resize Text and Reflow are both still independently satisfied, right?

If so, how about an explicit note like the suggestion above?

Reply:
As Reflow is motivated by zooming 400%, where the understanding text states if the text increases to 200% while zooming, I am afraid this remark might be confusing. Does it talk about increasing the text size starting from normal text size on a 320px wide device? I feel that falls under txt resizing. The text resizing SC does not talk about device size, which implies, the support needs to be granted on the device(s) supported.
The text resizing SC allow tow-dimensional zooming anyway.

I see your motivation. Can we word the snetence differently? 
Like "Supporting Text resizing on a 320 CSS px wide device goes beyond the Reflow requirement. As a consequence, introducing two-dimensional scrolling while resizing text to 200% on a 320 CSS px wide device does not collide with Reflow-conformance."

Reply:
I like Gundula's suggestion, with the exception of the term "collide", which sounds a little odd in this context.

I think "combine" would work? Or perhaps "conflict"?

Reply:
i took the gist of gundula's suggestion, and generalized it a bit - so it's not just about having a specific 320px viewport and then zooming in - but _any_ default smaller viewport that when zoomed in, becomes smaller than specific sizing requirements of reflow.  

it's far more likely that people's smaller devices will not neatly match 320px, after all.

Reply:
Wonderful, [this specific paragraph in the commit](https://github.com/w3c/wcag/pull/4055/commits/bd7a63a210e334a5fea707af3a67bd0915b59989#:~:text=If%20zooming%20in%20a%20web%20page%20by%20200%25%20results%20in%20a%20viewport%20that%20is%20smaller%20than%20the%20sizing%20requirements%20of%20Reflow%2C%20then%20two%2Ddimensional%20scrolling%20may%20occur.%20This%20would%20not%20be%20a%20failure%20of%20Reflow%2C%20as%20the%20sizing%20of%20the%20viewport%20would%20be%20smaller%20than%20what%20Reflow%20requires.) removes all doubt, and much more elegantly:

> If zooming in a web page by 200% results in a viewport that is smaller than the sizing requirements of Reflow, then two-dimensional scrolling may occur. This would not be a failure of Reflow, as the sizing of the viewport would be smaller than what Reflow requires.

Thanks very much, @gundulaniemann and @scottaohara!